docs/_spec/03-types.md

```diff
@@ -582,10 +597,13 @@ The conformance relation ´(<:)´ is the smallest transitive relation that satis
 - A singleton type `´p´.type` conforms to the type of the path ´p´.
 - A singleton type `´p´.type` conforms to the type `scala.Singleton`.
 - A type projection `´T´#´t´` conforms to `´U´#´t´` if ´T´ conforms to ´U´.
-- A parameterized type `´T´[´T_1´, ..., ´T_n´]` conforms to `´T´[´U_1´, ..., ´U_n´]` if the following three conditions hold for ´i \in \{ 1, ..., n \}´:
+- A parameterized type `´T´[´T_1´, ..., ´T_n´]` conforms to `´T´[´U_1´, ..., ´U_n´]` if the following conditions hold for ´i \in \{ 1, ..., n \}´:
   1. If the ´i´'th type parameter of ´T´ is declared covariant, then ´T_i <: U_i´.
   1. If the ´i´'th type parameter of ´T´ is declared contravariant, then ´U_i <: T_i´.
   1. If the ´i´'th type parameter of ´T´ is declared neither covariant nor contravariant, then ´U_i \equiv T_i´.
+  1. If ´U_i´ is a wildcard type argument of the form ´\\_ >: L <: U´, then ´T_i >: L´ and ´T_i <: U´.
+  1. If ´U_i´ is a wildcard type argument of the form ´\\_ >: L_1 <: U_1´ and ´T_i´ is a wildcard argument of the form ´\\_ >: L_2 <: U_2´, then ´L_1 <: L_2´ and ´H_2 <: H_1´.
+  1. If ´U_i´ is a wildcard type argument of the form ´\\_ >: L <: U´ and the ´i´'th type parameter is declared covariant, then 
```

Comment:
Looks like the sentence was not finished here. Also, the contravariant case is missing.